demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -1,4 +1,5 @@
 import copy
+from copy import deepcopy
```

Comment:
nice

Reply:
I actually got rid of it now and fixed the cache clearing mechanism of the `get_file` function to work properly and used that instead (per @yaakovi's recommendation).

Reply:
that's disappointing